docs/vm-installation.md

```diff
@@ -0,0 +1,159 @@
+
+```bash
+AGENT_NAMESPACE="vm-ns"
+AGENT_CONF_PATH="/etc/antrea"
+mkdir -p $AGENT_CONF_PATH
+# Copy antrea-agent kubeconfig files
+cp ./antrea-agent.kubeconfig $AGENT_CONF_PATH
+cp ./antrea-agent.antrea.kubeconfig $AGENT_CONF_PATH
+# Update clientConnection and antreaClientConnection
+sed -i "s|kubeconfig: |kubeconfig: $AGENT_CONF_PATH/|g" antrea-agent.conf
+sed -i "s|#externalNodeNamespace: default|externalNodeNamespace: $AGENT_NAMESPACE|g" antrea-agent.conf
+# Copy antrea-agent configuration file
+cp ./antrea-agent.conf $AGENT_CONF_PATH
+```
+
+3. Create `antrea-agent` service. Below is a sample snippet to start `antrea-agent` as a service on Ubuntu 18.04 or
+   later:
+
+```bash
+AGENT_BIN_PATH="/usr/sbin"
+AGENT_LOG_PATH="/var/log/antrea"
+mkdir -p $AGENT_BIN_PATH
+mkdir -p $AGENT_LOG_PATH
+cat << EOF > /etc/systemd/system/antrea-agent.service
+Description="antrea-agent as a systemd service"
+After=network.target
+[Service]
+ExecStart=$AGENT_BIN_PATH/antrea-agent \
+--config=$AGENT_CONF_PATH/antrea-agent.conf \
+--logtostderr=false \
+--log_file=$AGENT_LOG_PATH/antrea-agent.log
+Restart=on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable antrea-agent
+sudo systemctl start antrea-agent
+```
+
+## Installation on Windows VM
+
+### Prerequisites
+
+1. Enable the Windows Hyper-V optional feature on Windows VM.
+
+```powershell
+Install-WindowsFeature Hyper-V-Powershell
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All -NoRestart
+```
+
+2. OVS needs to be installed on the VM. For more information about OVS installation please refer to the [Antrea Windows documentation](windows.md#1-optional-install-ovs-provided-by-antrea-or-your-own).
+3. Download [nssm](https://nssm.cc/download) which will be used to create the Windows service for `antrea-agent`.
+
+Note: Only Windows Server 2019 is supported in the first release at the moment.
+
+### Installation
+
+1. Build `antrea-agent` binary in the root of the antrea code tree and copy the `antrea-agent` binary from the `bin`
+   directory to the Windows VM.
+
+```bash
+#! /bin/bash
+make docker-windows-bin
+```
+
+2. Copy `antrea-agent.conf`, `antrea-agent.kubeconfig` and `antrea-agent.antrea.kubeconfig` files to the VM. Please
+   refer to the step 2 of [Installation on Linux VM](vm-installation.md#installation) section for more information.
+
+```powershell
+$WIN_AGENT_CONF_PATH="C:\antrea-agent\conf"
+New-Item -ItemType Directory -Force -Path $WIN_AGENT_CONF_PATH
+# Copy antrea-agent kubeconfig files
+Copy-Item .\antrea-agent.kubeconfig $WIN_AGENT_CONF_PATH
+Copy-Item .\antrea-agent.antrea.kubeconfig $WIN_AGENT_CONF_PATH
+# Copy antrea-agent configuration file
+Copy-Item .\antrea-agent.conf $WIN_AGENT_CONF_PATH
+```
+
+3. Create `antrea-agent` service using nssm. Below is a sample snippet to start `antrea-agent` as a service:
+
+```powershell
+$WIN_AGENT_BIN_PATH="C:\antrea-agent"
```

Comment:
Do we have a better path for config and bin files, than "C:\"?

Reply:
On Windows, the binary files are always placed under path "C:\Program Files", while the data files like logs and configurations are placed under path "C:\ProgramData", e.g., we could have "C:\Program Files\antrea\antra-agent.exe", "C:\ProgramData\antrea\etc\antrea-agent.conf" and "C:\ProgramData\antrea\logs". Not strong opinion on using both "C:\Program Files" and "C:\ProgramData". But I would prefer to have a variable to describe the home path for antrea files if placing all files in one root directory, like "$ANTREA_AGENT_HOME=C:\antrea-agent"

Reply:
IMO it would be easier to follow, if the required files are in one location. Here the user is copying the files manually. 

Having one folder, the end-user may not pay attention to conf files. So I thought of having conf and logs directory in the C:\antrea-agent and let the user copy the files accordingly. 

Having said that, I'm fine to change the doc instructions as per the needs.

PS C:\antrea-agent> ls
    Directory: C:\antrea-agent

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----        7/18/2022   5:42 AM                conf
d-----        7/18/2022   5:45 AM                logs
-a----        7/18/2022   5:41 AM       83733504 antrea-agent.exe